soc/arm/arm/mps2/dts_fixup.h

```diff
@@ -36,6 +36,11 @@
 #define DT_CMSDK_AHB_GPIO3			DT_ARM_CMSDK_GPIO_40103000_BASE_ADDRESS
 #define DT_IRQ_PORT3_ALL			DT_ARM_CMSDK_GPIO_40103000_IRQ_0
 
+#define DT_VDEV_START_BASE_ADDRESS		DT_MMIO_SRAM_28180000_BASE_ADDRESS
+#define DT_VDEV_SIZE				DT_MMIO_SRAM_28180000_SIZE
+
+#define DT_IPM_DEV				DT_ARM_MHU_40003000_LABEL
+
```

Comment:
With overlays in sample you need to use a chosen{} defined in the overlay instead
(https://docs.zephyrproject.org/latest/guides/dts/index.html?highlight=chosen#aliases-and-chosen-nodes)

Reply:
@karl-zh  
I updated my PR (https://github.com/zephyrproject-rtos/zephyr/pull/16985). I implemented the DT overlay in the sample with the use of chosen{} to have a generic name for the shared memory pool.  To do similar implementation, you have to get this commit : https://github.com/zephyrproject-rtos/zephyr/pull/16985/commits/19309c7a5608b6c2b6f06dab9fceb9edba8a9acf. Then you have to create the overlay using the template <board>.overlay( including the memory and the chosen declarations ). The overlay will be automatically applied on top of and DT_IPC_SHM_BASE_ADDRESS + DT_IPC_SHM_BASE_SIZE defines will be generated.

@galak could you confirm if you are in line with this DT overlay approach